daemon/MenuDaemon.vala

```diff
@@ -258,5 +268,65 @@ namespace Gala {
                 push_in = true;
             }, 3, Gdk.CURRENT_TIME);
         }
+
+        private void init_desktop_menu () {
+            change_wallpaper_accellabel = new Granite.AccelLabel (_("Change Wallpaper…"));
+
+            change_wallpaper = new Gtk.MenuItem ();
+            change_wallpaper.add (change_wallpaper_accellabel);
+            change_wallpaper.activate.connect (() => {
+                try {
+                    AppInfo.launch_default_for_uri ("settings://desktop/appearance/wallpaper", null);
+                } catch (Error e) {
+                    warning ("Failed to open wallpaper settings: %s", e.message);
```

Comment:
We should probably throw a dialog if these launches fail and not fail silently

Reply:
What would the corpus be?

Reply:
I did some dialogs